test/dataframe.jl

```diff
@@ -1,31 +1,52 @@
-using DataArrays,DataFrames
+using NullableArrays,CategoricalArrays,DataFrames
 
-v110 = rcopy(DataArray,reval("x <- 1:10"))
-@test isa(v110,DataVector)
-@test eltype(v110) == Cint
+@test rcopy(Nullable, RObject(1)).value == 1
+@test rcopy(Nullable, RObject("abc")).value == "abc"
+@test rcopy(RObject(Nullable(1))) == 1
+@test rcopy(Nullable, RObject(Nullable(1, true))).isnull
+
+v110 = rcopy(NullableArray,reval("c(1L, NA)"))
+@test isa(v110,NullableVector)
+@test eltype(v110) == Nullable{Int32}
+@test rcopy(NullableArray, RObject(v110[2]))[1].isnull
 
 attenu = rcopy(DataFrame,:attenu)
 @test isa(attenu,DataFrame)
 @test size(attenu) == (182,5)
 
 dist = attenu[:dist]
-@test isa(dist,DataArray{Float64})
+@test isa(dist,Vector{Float64})
+station = attenu[:station]
+@test isa(station,NullableCategoricalArray)
 
-@test rcopy(DataArray,"c(NA,TRUE)").na == @data([NA,true]).na
-@test rcopy(DataArray,"c(NA,1)").na == @data([NA,1.0]).na
-@test rcopy(DataArray,"c(NA,1+0i)").na == @data([NA,1.0+0.0*im]).na
-@test rcopy(DataArray,"c(NA,1L)").na == @data([NA,one(Int32)]).na
-@test rcopy(DataArray,"c(NA,'NA')").na == @data([NA,"NA"]).na
-@test_throws ErrorException rcopy(DataArray,"as.factor(c('a','a','c'))")
-@test rcopy(PooledDataArray,"as.factor(c('a','a','c'))").pool == ["a","c"]
+@test rcopy(NullableArray,"c(NA,TRUE)").isnull == NullableArray([true,true], [true,false]).isnull
```

Comment:
I meant isequal(rcopy(NullableArray,"c(NA,TRUE)"), NullableArray([true,true], [true,false])` and so on (i.e. convert both sides to a `NullableArray`).